flake.nix

```diff
@@ -4,7 +4,7 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     flake-parts = { url = "github:hercules-ci/flake-parts"; inputs.nixpkgs-lib.follows = "nixpkgs"; };
-    microvm = { url = "github:astro/microvm.nix"; inputs.nixpkgs.follows = "nixpkgs"; inputs.spectrum.follows = ""; };
+    microvm = { url = "github:sarcasticadmin/microvm.nix/rh/1707108673virtio"; inputs.nixpkgs.follows = "nixpkgs"; inputs.spectrum.follows = ""; }; # Currently using this fork since the upstream seems to be causing an issue
```

Comment:
we could be more explicit but since theres the accompanying `flake.lock` I dont think this will be an issue.

Im also going to raise it with upstream so we couldnt have to worry about this for much longer

Reply:
ah yea, `flake.lock`... :man_facepalming:. Thanks @sarcasticadmin